src/coprocessor/endpoint.rs

```diff
@@ -411,7 +290,7 @@ impl ReqContext {
 #[derive(Debug)]
 struct RequestTracker {
     running_task_count: Option<Arc<AtomicUsize>>,
-    ctx_pool: Option<CopContextPool>,
+    ctx_pool: Option<futurepool::ContextDelegators<ReadPoolContext>>,
```

Comment:
`ContextDelegator` is already taken. `ContextDelegator` and `ContextDelegators` are in fact different things in the `futurepool`. We can make a type alias though, but the type is only used once (i.e. here) so that I didn't do it.